Tests/SwiftCliCoreTests/ChartifyTest.swift

```diff
@@ -84,9 +93,7 @@ class MockFileValidator: FileValidator {
 class MockFileWriter: FileWriter {
     var wasWriteFileCalled = false
 
-    override func writeFile(chart: String,
-                            filePath: String,
-                            fileName: String) {
+    override func writeOutput(output: String) {
 
         wasWriteFileCalled = true
```

Comment:
Oh, I missed it during pairing, but let's update this to `wasWriteOutputCalled`

Reply:
And actually, we can call it a `MockOutputWriter: OutputWriter` since `Chartify` doesn't know or care anymore whether it's a `FileWriter` or not.

Reply:
Done!

Reply:
This was interesting, rather than creating a mock with an `override`  I ended up just making a generic instance of the `OutputWriter` prototype. Is that okay?

Reply:
Yup, that's the right thing to do!